tests/Jwt/FirebaseParserTest.php

```diff
@@ -51,32 +50,15 @@ protected function setUp()
         ];
     }
 
+    /**
+     * @expectedException \Equip\Auth\Exception\InvalidException
```

Comment:
The referenced feature is [specific to PHPUnit 5.2](https://github.com/sebastianbergmann/phpunit/wiki/Release-Announcement-for-PHPUnit-5.2.0#new-features), which requires PHP >= 5.6. equip/auth currently still supports PHP 5.5 (see [Travis](https://github.com/equip/auth/blob/8e3fe70d4c44a362746ac6964f83a8c4e380107d/.travis.yml#L4) and [Composer](https://github.com/equip/auth/blob/8e3fe70d4c44a362746ac6964f83a8c4e380107d/composer.json#L16)). As such, in order to implement this, we'd have to drop 5.5 support.